documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1378,17 +1378,25 @@ This limitation will be removed in the next {prodname} release.
 ====
 
 |[[mongodb-property-mongodb-connection-string]]<<mongodb-property-mongodb-connection-string, `+mongodb.connection.string+`>>
-|
-|Mongodb connection string used for initial replica set discovery. This option requires `mongodb.members.auto.discover` to be set to `true` and it mustn't be used together with `mognodb.hosts` +
+|No default
+|Specifies a connection string that the connector uses during the initial discovery of a MongoDB replica set.
+To use this option, you must set the value of `mongodb.members.auto.discover` to `true`.
+Do not set this property and the `mongodb.hosts` property at the same time. +
  +
 [NOTE]
 ====
-At the moment the connection string is used only for the initial replica set discovery. Other
-configuration properties are still used when connecting directly to primary rs members -- with the exception of credential which (when present) is extracted from given connection string.
+The connector uses this connection string only during the initial replica set discovery process.
+During this discovery process, the connector ignores connection values that are specified in other properties (`mongodb.user`, `mongodb.password`, `mongodb.authsource`, SSL configuration properties, and so forth).
+After the discovery process completes, when the connector attempts to establish a direct connection to a primary replica set member, the connector then returns to using the standard connection properties. +
+ +
+When establishing a connection, the connector generally ignores this connection string.
```

Comment:
This and the consequent lines implies that in some cases, the connector might use properties from connection string (unless provided directly with the regular connector properties). However at the moment this is true strictly for credential properties.

Reply:
@jcechace Ok, I think that we're on the same page, but I need to reword this. What I mean is that, except during the discovery phase, the connector ignores the connection string, unless it doesn't find credential information elsewhere in the configuration. In that case, it will extract the credentials from the connection string.

Reply:
@jcechace Can you confirm whether it is correct that, in establishing a connection to a replica set member, the connector will extract credentials from the connection string if the credentials are not otherwise present in the configuration?

Reply:
@jcechace I went ahead and made an update based on how I interpreted your feedback from a couple weeks ago.  Please take a look when you get a chance and let me know if any further changes are necessary.